examples/c/sockfilter.bpf.c

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
+/* Copyright (c) 2022 Jacky Yin */
+#include <linux/bpf.h>
+#include <linux/if_ether.h>
+#include <linux/ip.h>
+#include <linux/in.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_endian.h>
+#include <stddef.h>
+#define IP_MF		0x2000
```

Comment:
nit: please add empty line above to separate includes from defines

Reply:
Fixed it too.